activities/finance-management/reporting.md

```diff
@@ -0,0 +1,45 @@
+---
+type: Guide
+explains: reporting requirements
+---
+
+# Reporting
+
+## Board meeting reports
+
+A concise financial one-pager is produced by the Operations Coordinator for the bi-monthly board meeting. The one-pager gives an overview of:
+
+* The current cash position
+* Costs incurred
+* Runway
+* General information (depending on time of the year, e.g. updates on (new) funding sources, important budget changes and interim conclusion)
+
+## Annual reporting
+
+Annual reporting consists of the annual financial statement and an annual report. The latter is a powerful piece of communications that shows how we deliver on our mission, and celebrate our accomplishments.
+
+## Financial annual statement
+
+Our first financial year runs from 1 July 2019 - 31 December 2020. The Operations Coordinator is working closely with the bookkeeper to produce the statement. An ANBI has to publish the financial statement on our website. The financial statement may concern the following information:
+
+* The balance sheet
+* The profit and loss statement
+* An additional explanation
+
+The financial statement must be published on our website within 6 six months after the end of the financial year.
+
+[Template for our annual financial statement.](https://drive.google.com/drive/folders/1eeW-cCescwdGoxuCKwB4oMlhE5512Qu_)
+
+## King Baudouin Foundation United States (KBFUS)
+
+The King Baudouin Foundation’s mission is to contribute to a better society. The Foundation is an actor for change and innovation, serving the public interest and increasing social cohesion in Belgium and Europe. They seek to maximize their impact by strengthening the capacity of organizations and individuals. They also stimulate effective philanthropy by individuals and corporations. The Foundation’s key values are integrity, transparency, pluralism, independence, respect for diversity, and promoting solidarity. The Foundation’s current areas of activity are social justice & poverty, health, heritage and culture, social engagement, Africa, Latin America, Asia, Education and development of talents, Europe, climate, the environment and biodiversity. The King Baudouin Foundation is a public benefit foundation.
```

Comment:
```suggestion
```

Reply:
No need to include this information about them - this section is about reporting and what we need to deliver based on our legal agreements with them. Providing the link to the drive with all the background info should be sufficient.